registration/tests/utils/test_utils.py

```diff
@@ -22,6 +22,7 @@ def test_create_test_family(self):
         )
 
     def test_create_test_parent(self):
+        utils.create_test_fields()
```

Comment:
could we move this to `setUp` instead? then we can update the `create_test_fields` test to:
```
def test_create_test_fields(self):  # original name was not great oops
  Field.objects.all().delete()
  # rest of test
```

Reply:
wait, the test doesn't run when it's named `create_test_fields` but when renamed to `test_create_test_fields` it runs .. any idea why?

Reply:
the naming convention for the unittest framework needs test names to start with `test`! I missed it on the original one :')

Reply:
all good. Thanks for the info!